hetio/hetnet.py

```diff
@@ -455,7 +455,15 @@ def get_id(self):
         )
         return metaedge_id
 
+    @property
+    def abbrev(self):
+        return self.get_abbrev()
+
     def get_abbrev(self):
+        """
+        Getter function for a metaedge's abbreviation, kept for backwards
```

Comment:
If this is just kept for backwards compatibility, then shouldn't `abbrev` have the function and `get_abbrev` call `abbrev`?